lib.sh

```diff
@@ -37,7 +37,7 @@ if [[ "$OS_RELEASE_ID" == "ubuntu" ]]; then
     SUDO="$SUDO env DEBIAN_FRONTEND=$DEBIAN_FRONTEND"
 fi
 
-if [[ -d "/usr/share/automation" ]]; then
+if [[ -r "/etc/automation_environment" ]]; then
```

Comment:
I don't know if this is important/relevant, but there's still one instance of `/usr/share/automation` in the repo:

https://github.com//containers/automation_images/blob/752b774bb3e29fb11e43890f07fd978d8d1eac1e/lib.sh#L63

Reply:
Ya this is fine because that function specifies `INSTALL_PREFIX=/usr/share`.  I only changed it here because `INSTALL_PREFIX` could change, but `/etc/automation_environment` will always be a constant indicator that the library/tooling has been installed.